### ISSUE TITLE
Unable to override body-parser settings

### ISSUE THREAD

**Issue by Dids** [realm/realm-object-server#320]
## Goals

I'm trying to increase the upload size of POST and PUT requests in a custom `Service`, so using the built-in `express` app of ROS.

## Expected Results

I should be able to override body-parser settings, so I could upload larger payloads.

## Actual Results

Overriding body-parser settings has zero effect.

## Steps to Reproduce

Try to add/use a modified express body-parser, then upload a file or JSON content larger than 10k characters (or roughly 1 MB, I'm assuming).

Server response:

```json
{"title":"request entity too large","status":413}
```

## Code Sample

```js
...
this.middlewares.push(bodyParser.json({ limit: '50mb' }))
this.middlewares.push(bodyParser.urlencoded({ extended: true, limit: '50mb', parameterLimit: 50000 }))
...
return this.server.start({
  ...
  middlewares: this.middlewares
})
...
```

NOTE: I've also tried to override the middlewares after startup, using `this.server.expressApp.use()`, but that produces the same exact results.

## Version of Realm and Tooling

- Realm Object Server Version: 2.4.2
- Flavor:
  - [x] Developer
  - [ ] Professional
  - [ ] Enterprise
- Server OS & Version: Ubuntu 16.04
- Client SDK Version: N/A
- Client OS & Version: N/A

## Logs

Nothing is generated in the logs (even with log level set to `all`).

**Comment by Dids** [comment]
Commenting out line 117 in `Server.js` "fixes" the issue, as it looks like the body-parser added by ROS takes precedence.

**Comment by Dids** [comment]
As a temporary workaround, I'm removing the ROS added body-parser after startup, like so:

```js
for (let i in this.server.expressApp._router.stack) {
  let layer = this.server.expressApp._router.stack[i]
  if (layer.name === 'jsonParser') {
    this.server.expressApp._router.stack.splice(i, 1)
    break
  }
}
```

**Comment by Dids** [comment]
@nirinchev Was this implemented? Or not going to be implemented at all?

**Comment by nirinchev** [comment]
Sorry - it's in a private repo, which is why I guess Github didn't create a publicly visible comment. It was implemented and will be released with the next version. You'll be able to control it by setting:

```ts
server.start({
  // Other options
  jsonBodyLimit: '10mb'
})
```

**Comment by Dids** [comment]
@nirinchev I'm assuming this has been released now, since ROS was recently updated on npm?

**Comment by nirinchev** [comment]
It is, yes - it shipped with 2.5.0: https://github.com/realm/realm-object-server/blob/master/CHANGELOG.md#enhancements-1
